doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -132,6 +132,12 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+  fast_pair:
+    Initial Pairing: BT_FAST_PAIR
+    Subsequent pairing: BT_FAST_PAIR && EXPERIMENTAL
+    Battery Notification extension: BT_FAST_PAIR && EXPERIMENTAL
```

Comment:
Do we need to explicitly add `EXPERIMENTAL` here? `CONFIG_BT_FAST_PAIR` selects experimental already.

Reply:
Yes, but we want to remove the EXPERIMENTAL selection from the BT_FAST_PAIR while still tagging the BN extension as experimental.

Reply:
So why don't we add experimental to BT_FAST_PAIR_PN? Or it will be done in Kconfig.fast_pair?